subgraphs/uniswap-forks/protocols/pangolin/src/common/constants.ts

```diff
@@ -2,7 +2,7 @@
 //////Versions//////
 ////////////////////
 
-export const PROTOCOL_SUBGRAPH_VERSION = "1.0.0";
+export const PROTOCOL_SUBGRAPH_VERSION = "1.0.4";
```

Comment:
Since you are at it, do you mind removing this constant and using instead versions.ts?

Reply:
Good idea! done.